app/AcquireIntersonArrayRF.cxx

```diff
@@ -155,6 +155,10 @@ int main( int argc, char * argv[] )
   container.SetNewRFImageCallback( &pasteIntoImage, &clientData );
 
   /* Begin theoretically unnecessary yet definitely necessary lines...*/
+  /**
```

Comment:
These are commented out now. So, they are theoretically unnecessary and practically unnecessary?